projects/pysan/pysan/pysan/sanlib.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2022 Google LLC
```

Comment:
Can you put this pysan code into somewhere in `infra/sanitizers` instead?

Reply:
Done